sdk/metricsadvisor/Azure.AI.MetricsAdvisor/src/Models/Metrics/MetricAnomalyFeedback.cs

```diff
@@ -14,12 +16,10 @@ internal partial class AnomalyFeedback : MetricFeedback
         /// <param name="endTime"> The end timestamp of feedback timerange, when equals to startTime means only one timestamp. </param>
         /// <param name="value"> The value for the feedback. </param>
         /// <exception cref="ArgumentNullException"> <paramref name="dimensionFilter"/> or <paramref name="value"/> is null. </exception>
-        public AnomalyFeedback(Guid metricId, FeedbackDimensionFilter dimensionFilter, DateTimeOffset startTime, DateTimeOffset endTime, AnomalyValue value) : base(metricId, dimensionFilter)
+        public MetricAnomalyFeedback(string metricId, FeedbackDimensionFilter dimensionFilter, DateTimeOffset startTime, DateTimeOffset endTime, AnomalyValue value) : base(metricId, dimensionFilter)
         {
-            if (dimensionFilter == null)
-            {
-                throw new ArgumentNullException(nameof(dimensionFilter));
-            }
+            Argument.AssertNotNull(dimensionFilter, nameof(dimensionFilter));
```

Comment:
No need to validate `dimensionFilter` here, since the base class validates it (or it should).